charts/testkube-operator/templates/webhook.yaml

```diff
@@ -40,5 +44,8 @@ webhooks:
     - UPDATE
     resources:
     - testtriggers
+    {{- if .Values.webhook.scope }}
+    scope: {{ .Values.webhook.scope }}
```

Comment:
`scope` can be hardcoded to `Namespaced` as all our resources are namespace-scoped